README.md

```diff
@@ -1,50 +1,65 @@
 # roboteam_suite
-All relevant roboteamtwente repositories in one.
+This repo contains:
+- Roboteam AI, which listens to Roboteam World and makes decisions on what the robots should do based on the vision data.
+- Roboteam World, which interprets vision data (received from GRSim, or from a physical camera)
+- Roboteam Robothub, used to send commands to the robots (either through GRSim or the base station)
+- Roboteam Utils, which contains a lot of helper functions, primarily geometrical in nature, such as Lines, Triangles, etc. These are used in other repo's, such as Roboteam AI and Roboteam World
+- Roboteam Proto, which contains the protocol used to send and receive messages over TCP/UDP 
 
+In order to test this code, we use the GRSim simulator developed by the SSL Team Parsian. However, due to differences in the control mechanism of the robot (specifically, theta vs omega control), this software requires the [RoboTeam fork of GRSim](https://github.com/RoboTeamTwente/grSim). Installation instructions can be found in the readme of that repo.
 
-# See also
-
-Running with camera's and real robots:
-SSL-Vision (https://github.com/RoboCup-SSL/ssl-vision)
-
-Working with a referee
-SSL-Game-controller (https://github.com/RoboCup-SSL/ssl-game-controller)
-SSL-vision-client (https://github.com/RoboCup-SSL/ssl-vision-client)
-
-Working with the grSim simulator
-grSim (https://github.com/RoboTeamTwente/grSim) 
+Please note:
+This software is tested on Ubuntu 18.04 and 20.04 LTS, and the installation instructions are written with that distro/OS in mind. However, other distro's of Linux are also likely to be able to run this software, albeit with some modifications.
 
 
 # Usage
-Easiest way to run things is to make some compound executables in CLion that boot everything you need. Make sure you have grsim or ssl-vision running and are listening to the right ports.
+First, clone this repository. We recommed using an SSH key if you are able to do so, as this immediately authorizes your future pushes to the repositories. 
+```
+git clone --recurse-submodule git@github.com:RoboTeamTwente/roboteam_suite.git
+```
+Now, you can open the roboteam_suite project with an IDE of your choice. We recommed CLion, as it has a lot of very helpful features and you can get a student license for it. 
+
+Make sure you have CMake installed. Go the CMakeLists.txt and click on it. CLion will generate a pop-up saying "project files not loaded" in blue in the top right corner. Clicking it will have CMake load the files. This means it generates instructions on how the files should be compiled in order to produce the executables specified in the CMakeLists.txt file. You can now compile the targets!
```

Comment:
```suggestion
Make sure you have CMake installed. Go to the CMakeLists.txt and click on it. CLion will generate a pop-up saying "project files not loaded" in blue in the top right corner. Clicking it will have CMake load the project. This means it generates instructions on how the files should be compiled in order to produce the executables specified in the CMakeLists.txt file. You can now compile the targets!
```